src/routes/jira/connect/enterprise/app/jira-connect-enterprise-app-router.ts

```diff
@@ -16,6 +16,9 @@ JiraConnectEnterpriseAppRouter.use("/:uuid", routerWithUUID);
 
 routerWithUUID.use(GithubServerAppMiddleware);
 routerWithUUID.route("")
-	.get(csrfMiddleware, JiraJwtTokenMiddleware, JiraConnectEnterpriseAppCreateOrEdit)
-	.put(JiraContextJwtTokenMiddleware, JiraConnectEnterpriseAppPut)
-	.delete(JiraContextJwtTokenMiddleware, JiraConnectEnterpriseAppDelete);
+	//eslint-disable-next-line @typescript-eslint/no-explicit-any
+	.get(csrfMiddleware, JiraJwtTokenMiddleware as any, JiraConnectEnterpriseAppCreateOrEdit)
```

Comment:
This is interesting, express typings enforce the `res.locals` has same type across the handlers within same route. And doesn't not distinguish `input type` and `output type` of `res.locals`.

I don't want to attach the type `gitHubAppConfig` to `JiraJwtTokenMiddleware` because it is use somewhere else, while `JiraConnectEnterpriseAppCreateOrEdit` required that `gitHubAppConfig` in `res.locals` for typings.

So just type it to `any` to convert it.

-----

So the way express typings enforce me, make me wonder is it a code smell the way we use `res.locals`?